crowdin.yml

```diff
@@ -0,0 +1,6 @@
+files:
+  - source: /messages/en.json
+    translation: /messages/%two_letters_code%.json
+  - source: '/public/metadata/messages/pt-BR.json'
+    translation: '/public/metadata/messages/%locale%.json'
+    type: 'json'
```

Comment:
```suggestion
    type: 'json'
     - source: "/public/metadata/licenses/pt-BR.md"
    translation: "/public/metadata/licenses/%locale%.md"
    type: "md"
```

Reply:
why `md`? this `template` will be used for future apps.